exonum/src/blockchain/mod.rs

```diff
@@ -433,8 +445,15 @@ impl Blockchain {
         fork.checkpoint();
 
         let catch_result = panic::catch_unwind(panic::AssertUnwindSafe(|| {
-            let context = TransactionContext::new(&mut *fork, &raw);
-            tx.execute(context)
+            if service_enabled(service_name, fork) {
+                let context = TransactionContext::new(&mut *fork, &raw);
+                tx.execute(context)
+            } else {
+                Err(ExecutionError::with_description(
```

Comment:
What if the service uses an error with code 255? Should we document somewhere that 255 is a reserved code? Maybe, it could make sense to handle this error by analogy with panics (i.e., by introducing a new variant in `TransactionErrorType`)?